lib/std/start.zig

```diff
@@ -282,6 +282,13 @@ fn _start() callconv(.Naked) noreturn {
             \\ and sp, #-16
             \\ b %[posixCallMainAndExit]
             ,
+            .riscv32 =>
+            \\ li s0, 0
+            \\ li ra, 0
+            \\ sw sp, %[argc_argv_ptr]
+            \\ andi sp, sp, -16
```

Comment:
I'm not sure if this is needed on riscv32 too.

Reply:
Seems like yes: https://github.com/riscv-non-isa/riscv-elf-psabi-doc/blob/2092568f7896ceaa1ec0f02569b19eaa42cd51c9/riscv-elf.adoc#tag_riscv_stack_align-4-uleb128value